src/main/java/org/spongepowered/api/block/BlockType.java

```diff
@@ -50,7 +51,7 @@
      *
      * @return The id
      */
-    String getId();
+    String getName();
```

Comment:
What is the id of the block then? The numerical id?
Because there is only a lookup method in the game registry for getById and not for getByName anymore.
(The javadocs may be misleading now)